File: BCLIQ.py
N = int(input())
A = range(0,N)
s = raw_input()
s = s+" "
k = 0
i = 0
while k<len(s) and i<N:
    temp = ""
    while s[k]==' ':
        k = k+1
    while s[k]!=' ':
        temp = temp+s[k]
        k = k+1
    A[i] = int(temp)
    i = i+1

del i
del k
del s
del temp
count = 1
Max = 1
for i in range(0,N-1):
    temp = A[i]
    for j in range(i+1,N):
        if temp<A[j]:
            count = count + 1
            temp = A[j]
    if count>Max:
        Max = count
    count = 1
print Max
    

File: BCLIQ.cpp
#include<stdio.h>

int main(void)
{
	int N,A[1000];
	scanf("%d",&N);
	for(int i=0;i<N;i++)
		scanf("%d",&A[i]);
	int dem=1,max=1;
	for(int i=0;i<N-1;i++)
	{
		int temp=A[i];
		for(int j=i+1;j<N;j++)
		{
			if(temp<A[j])
			{
				dem++;
				temp=A[j];
			}
		}
		if(dem>max)
			max=dem;
		dem=1;
	}
	printf("%d",max);
 return 0;
}

